scan/src/utils/HexUtils.re

```diff
@@ -1,10 +1,13 @@
-let normalizeHexString = hexstr => {
-  let len = hexstr->String.length;
-  let prefix = len >= 2 ? hexstr->String.sub(0, 2) : "";
-
-  switch (prefix) {
-  | "0x"
-  | "0X" => hexstr->String.lowercase_ascii->String.sub(2, len - 2)
-  | _ => hexstr->String.lowercase_ascii
-  };
-};
+// This function receive any string and fliter the latest part
```

Comment:
Make sure the sentence is grammatically correct and contains no spelling errors. At least you should pass it through grammarly.